.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+- id: clj-kondo
+  name: clj-kondo
+  description: A linter for Clojure code that sparks joy
+  entry: --entrypoint clj-kondo borkdude/clj-kondo:latest --lint
```

Comment:
Ideally this would point a specific version, but there is no Docker image that is tagged for the current release.